homeassistant/components/starlink/binary_sensor.py

```diff
@@ -0,0 +1,50 @@
+"""Contains binary sensors exposed by Starlink integration."""
+from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from ...helpers.entity import DeviceInfo
```

Comment:
Don't use relative paths outside your own integration package.

Reply:
Good catch! I've noticed sometimes VS Code will use relative imports for homeassistant modules, I wonder why it chose to do it here